OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs

```diff
@@ -297,27 +335,66 @@ private async void UpdateInitialVolume(object sender, MediaPlayerVolumeChangedEv
             // Updating the GUI should happen on the dispatcher.
             await App.Current.UIDispatcher.RunAsync(CoreDispatcherPriority.High, () =>
             {
-                MediaPlayer.VolumeChanged -= UpdateInitialVolume;
+                _mediaPlayer.VolumeChanged -= UpdateInitialVolume;
                 MediaVolumeLevel = (int)App.Current.UserSettings.Values["MediaVolume"];
                 IsBackBtnEnabled = true;
             });
         }
 
+        /// <summary>
+        /// Media playing event handler. Gets called when the media player state is changed to playing.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
         private async void MediaPlayer_Playing(object sender, EventArgs e)
         {
             Debug.WriteLine(DateTime.Now.ToString("hh:mm:ss.fff") + ": Media is playing");
-            await App.Current.UIDispatcher.RunAsync(CoreDispatcherPriority.High, () =>
+            if (_isFirstPlaying || _isReloading)
```

Comment:
I'd change this for a `_isLibvlcInitialized` private bool, which is false initially, and is inverted in `MediaPlayer_Playing` and `ReloadCurrentMedia`.

Currently, there is almost no functional difference between `_isFirstPlaying` and `_isReloading`, [other than here](https://github.com/fossmium/OneDrive-Cloud-Player/blob/c75747890a16727274a8939617e61444e5800603/OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs#L375) (which I'm not sure is intentional)

Reply:
It is done intentional as first time playing and reloading have different behaviors. as for the rename. I dont think that would be a improvement as libvlc gets initialized in the constructor and not in the media playing event. ReloadCurrentMedia can be used. But i like the explicit verbosity of `is` to indicate it being a boolean and not a command or some other sort of thing.

Reply:
Added a comment to clarify